testing/prow/gpu-operator.sh

```diff
@@ -357,18 +357,27 @@ prepare_cluster_for_gpu_operator_with_alerts() {
 }
 
 test_operatorhub() {
+    OPERATOR_NAMESPACE="nvidia-gpu-operator"
+
     if [ "${1:-}" ]; then
         OPERATOR_VERSION="--version=$1"
     fi
     shift || true
     if [ "${1:-}" ]; then
         OPERATOR_CHANNEL="--channel=$1"
+        if [[ "${OPERATOR_CHANNEL}" != *"1.9"* ]]; then
```

Comment:
Shouldn't this be `<=` ?

Reply:
version comparison doesn't work so simply, v1.10 would break it :(

it's not clear yet how they will handle channel naming in the future, so I prefer keeping it way, it will break when the next channels comes up and we'll find a proper fix then

Reply:
ack